INSTALL.md

```diff
@@ -81,17 +81,14 @@ GEOS_LIBRARY_PATH= '{YOUR PATH}/libgeos_c.1.17.0.dylib'
 
```

Comment:
There's some other bits of docs in this file that I reckon are a bit suspect.

Like do we really need all this stuff about GEOS/GDAL/PostGIS for this project? In CI we are just running plain old postgres. No GIS extensions at all.

Reply:
Also the setup instructions for importing data are b0rked

Reply:
https://github.com/DemocracyClub/WhoCanIVoteFor/pull/2040/commits/d339d3417471c2fb66581e2470cf55c6315b4756